DEVELOPMENT.md

```diff
@@ -27,7 +27,7 @@ To build Orbit you need a compiler capable of C++17. The following ones should b
 
 - GCC 8 and above on Linux
```

Comment:
Since you're already touching this file could you also replace this by `GCC 9 and above on Linux`? :innocent:

Reply:
Sure, done.